docs/examples/example.ipynb

```diff
@@ -236,27 +236,237 @@
+    "\n",
+    "To make this shell script executable, run this in the Terminal:\n",
+    "\n",
+    "```sudo chmod +x <FILEPATH>/exec.sh```\n",
+    "\n",
+    "### 5. Configure a modular resource\n",
+    "**NOTE:** *These instructions are for local development. Soon we will launch a `Registry` which will allow users to upload modules to app.viam.com directly. We will update the documentation with instructions on how to do that shortly.*\n",
+    "\n",
+    "[Configure your new module](https://docs.viam.com/extend/modular-resources/#configure-your-module) on your robot by navigating to the **Config** tab of the robot's page on the Viam app and click on **Modules** subtab. Add the name of your component and the executable path. Here, the path would be `<path-on-your-filesystem>/modular-arm/exec.sh`.\n",
+    "\n",
+    "Once you have configured a module as part of your robot configuration, [configure your modular resource](https://docs.viam.com/extend/modular-resources/#configure-your-modular-resource) made available by that module by adding new components or services configured with your modular resources' new type or model. To instantiate a new component from your module, specify the `type`, `model`, and `name` of your modular resource. This is a JSON example:\n",
+    "\n",
+    "```json\n",
+    "{\n",
+    "  \"components\": [\n",
+    "    {\n",
+    "      \"model\": \"acme:demo:myarm\",\n",
+    "      \"attributes\": {},\n",
+    "      \"depends_on\": [],\n",
+    "      \"name\": \"my-arm\",\n",
+    "      \"type\": \"arm\",\n",
+    "    }\n",
+    "  ],\n",
+    "  \"modules\": [\n",
+    "    {\n",
+    "      \"executable_path\": \"<path-on-your-filesystem>/modular-arm/exec.sh\",\n",
+    "      \"name\": \"modular_arm\",\n",
+    "    }\n",
+    "  ]\n",
+    "}\n",
+    "```\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create custom remotes\n",
+    "\n",
+    "While the main RDK is written in golang, you can create custom components in python and connect them to a robot as a `remote` component. This allows you to extend the functionality of a robot, or even create an entire robot exclusively in python.\n",
+    "\n",
+    "The steps required in creating a custom component and connecting it to the RDK are:\n",
+    "1. **Subclass a component and implement desired functions**. This will be your custom component.\n",
+    "    - For functions you do not wish to implement, you must at least define them by putting `pass` or `raiseNotImplementedError()` in the function.\n",
+    "1. **Create an `rpc.server.Server` instance and register the custom component**. This is the `RPC` server that enables communication with your custom component.\n",
+    "1. **Start the `Server` and register the running server as a remote**. Registering this `RPC` server as a remote allows the RDK to communicate with the server.\n",
```

Comment:
Is this the right place for a link to https://docs.viam.com/extend/custom-components-remotes/ ?

Reply:
This would be! I'll be working on this specific part of the docs in my next ticket, where we want the docs to be parallel with our regular docs.